### PR TITLE
new addon: Tvnamer

### PR DESCRIPTION
**Pull request by luivit**
tvnamer is a utility which to rename files from `some.show.s01e03.blah.abc.avi` to `Some Show - [01x03] - The Episode Name.avi` (by retrieving the episode name using data from tvdb_api).

It's my first addon for OpenELEC...I hope that everything is ok

### CODE REVIEW

File: addons/tools/tvnamer/changelog.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+6.0.0
```
Comment (sraue) [inline addons/tools/tvnamer/changelog.txt]: should be 7.0.0

File: addons/tools/tvnamer/package.mk [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+PKG_DEPENDS_TARGET="toolchain Python tvdb_api"
+PKG_PRIORITY="optional"
+PKG_SECTION="tools"
+PKG_SHORTDESC="Automatic TV episode namer/mover"
+PKG_LONGDESC="Automatically names downloaded/recorded TV-episodes, by parsing filenames and retrieving show-names from www.thetvdb.com."
+PKG_IS_ADDON="yes"
+PKG_ADDON_TYPE="xbmc.python.script"
+PKG_AUTORECONF="no"
+PKG_MAINTAINER="luivit/dbr"
+PKG_ADDON_REPOVERSION="6.0"
```
Comment (sraue) [inline addons/tools/tvnamer/package.mk]: this too, 7.0

**Comment by sraue** [comment]
other then that (without testing) looks good so far :-)

**Comment by luivit** [comment]
Done, thx :)

**Pull request merged**